### PR TITLE
hue lights devices

### PR DESCRIPTION
**Pull request by GwennanS**
Added a hue device and room configuration to test it

### CODE REVIEW

File: cc_library/src/scripts/hue_lights.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        if instruction == "scene":
+            self.set_scene(action)
+        if instruction == "manual":
+            self.set_manual(action.get("component_id"), action.get("value"))
+        else:
+            return False, action
+        return True, None
+
+    def test(self):
+        print("test")
```
Comment (IssaHanou) [inline cc_library/src/scripts/hue_lights.py]: print left

File: cc_library/src/scripts/hue_lights.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        requests.put(
+            "http://192.168.178.128/api/JQrPwJNthHtfPEG9vhW3mqwIVuFo3ESLD3gvkZOB/groups/Spotlights/action",
+            data=params,
+            headers=headers,
+        )
+        time.sleep(2)
+        params = json.dumps({"scene": self.scene})
+        headers = {"Content-type": "application/json"}
+        resp = requests.put(
+            "http://192.168.178.128/api/JQrPwJNthHtfPEG9vhW3mqwIVuFo3ESLD3gvkZOB/groups/Spotlights/action",
```
Comment (IssaHanou) [inline cc_library/src/scripts/hue_lights.py]: can we put this in a global variable

File: cc_library/src/scripts/hue_lights_config.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  "description": "4 spotlights",
+  "host": "localhost",
+  "labels": ["lamp"],
+  "port": 1883,
+  "input": {},
+  "output": {
+    "spot1": {
+      "type": "string",
+      "instructions": {
+        "scene": "string"
```
Comment (mrlomar) [inline cc_library/src/scripts/hue_lights_config.json]: should also have the manual instruction 4x

File: cc_library/src/scripts/hue_lights.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        self.scene = "none"
+        params = json.dumps({"on": True, "bri": 50, "xy": [0.3, 0.3]})
+        headers = {"Content-type": "application/json"}
+        resp = requests.put(
+            "http://192.168.178.128/api/JQrPwJNthHtfPEG9vhW3mqwIVuFo3ESLD3gvkZOB/groups/Spotlights/action",
+            data=params,
+            headers=headers,
+        )
+        if resp.status_code == 200:
+            print("Template has been published.")
```
Comment (IssaHanou) [inline cc_library/src/scripts/hue_lights.py]: can you use the logger

**Pull request merged**